examples/pokemon-service-client-usage/examples/trace-serialize.rs

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+/// This example demonstrates how an interceptor can be written to trace what is being
+/// serialized / deserialized on the wire.
```

Comment:
```suggestion
/// serialized / deserialized on the wire.
///
/// Please beware that this may log sensitive information! This example is meant for pedagogical purposes and may be useful in debugging scenarios. Please don't use this as-is in production.
```